packages/q/qt6-base/package.yml

```diff
@@ -54,8 +54,7 @@ rundeps    :
     - devel :
         - libxkbcommon-devel
         - mesalib-devel
-        - vulkan
-        - vulkan-headers
+        - vulkan-devel
```

Comment:
```suggestion
        - pkgconfig(vulkan)
```

Reply:
You actually can't use pkgconfig deps as rundeps

Reply:
Oh, haha, I totally missed that this was in the `rundep` section. Disregard! ^^